### DISCUSSION TITLE
How to add a text editor widget?

### DISCUSSION THREAD

**Discussion by GameTn** [atoti/atoti#544] [Q&A] [answered]
Hi , i think it will be nice if we can have text box to insert in the dashboard and why not images

**Comment by tibdex** [comment, marked as answer]
Hi, this is already available as an Atoti+ feature by passing the [`ADVANCED_APP_EXTENSION`](https://docs.atoti.io/0.6.5/lib/atoti-plus/atoti_plus.app_extension.html#atoti_plus.app_extension.ADVANCED_APP_EXTENSION) to [`Session()`'s *app_extensions* parameter](https://docs.atoti.io/0.6.5/lib/atoti/atoti.session.html#atoti.Session.__init__):

![Screen Shot 2022-03-30 at 13 21 02](https://user-images.githubusercontent.com/6574550/160823292-4bdaab58-2c58-4e73-abfb-31797e952b4d.png)
